src/configuration_env.js

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2018 Jaeger Author.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+
+import Configuration from './configuration.js';
+import Utils from './util.js';
+
+export default class ConfigurationEnv {
+  static _getConfigValue(obj, key, env) {
+    return obj ? obj[key] || env : env;
```

Comment:
this seems more intuitive: `return obj && obj[key] || env`. Also, I'd replace `env` with `defaultValue`

Reply:
That is more intuitive. Thanks!